source/iostreams.tex

```diff
@@ -11848,9 +11848,9 @@
 identifies the location of a file when resolved relative to
 an implied starting location. The elements of a path that determine if it is
 relative are operating system dependent.
-\begin{note}
-Pathnames ``.''\ and ``..''\ are relative paths.
-\end{note}
+\begin{example}
+Pathnames ``.''\ and ``..''\ are relative paths on POSIX and Windows systems.
```

Comment:
According to the grammar they are always relative paths.

Reply:
I think this sentence applies to the native format, whose grammar is OS-dependent.

Reply:
The preceding sentence says

> The elements of a path that determine if it is relative are operating system dependent.

So it does look like the standard allows an OS to treat `.` or `..` as a non-relative path.